output.R

```diff
@@ -21,6 +21,17 @@ options(error = quote({
   q()
 }))
 
+argv <- get0("argv", ifnotfound = commandArgs(trailingOnly = TRUE))
+
+# run updates before loading any packages
+if ("--update" %in% argv) {
+  stopifnot(`--update must not be used together with --renv=...` = !any(startsWith(argv, "--renv=")))
+  source("scripts/utils/updateRenv.R")
```

Comment:
CRAN packages should only be updated manually, or when a PIK package depends on a newer version, yes. It's similar to what we have now: PIK packages are always kept up-to-date and CRAN packages are updated manually or when a PIK package requires it. The main difference would be that updates are no longer shared, everyone has to update CRAN packages on their own.

Reply:
Did we speak about that already somewhere? Maybe that is the optimal policy, maybe not. Because "everyone has to update CRAN packages on their own" is a big change in policy. What we have now is (implicitly): "We'll do CRAN updates for you, don't worry." with a fuzzy definition of "we".

Reply:
Hm that's not how I understand our current policy, I'd say it's closer to "never update CRAN packages, unless really necessary". Anyway it's certainly very fuzzy, and I'm open for a discussion on this.

Reply:
After discussing with the RSE group again we decided to leave the CRAN package update strategy as it is, and readjust if outdated CRAN packages become a problem.